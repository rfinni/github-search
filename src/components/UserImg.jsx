import React from 'react';
import PropTypes from 'prop-types';

const UserImg = ({ user }) => (
  <div className="image-wrapper">
    <img className="user-image" src={user.avatar} alt={user.name} />
  </div>
);

UserImg.propTypes = {
  user: PropTypes.object.isRequired,
};

export default UserImg;
