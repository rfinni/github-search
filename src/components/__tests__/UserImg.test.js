import React from 'react';
import renderer from 'react-test-renderer';
import UserImg from '../UserImg';
import { stubUser } from '../../../testing/stubs';

describe('UserImg.jsx', () => {
  const wrapper = renderer.create(<UserImg user={stubUser} />).toJSON();
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

