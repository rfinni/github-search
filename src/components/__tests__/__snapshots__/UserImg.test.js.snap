// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserImg.jsx should render correctly 1`] = `
<div
  className="image-wrapper"
>
  <img
    alt="Ryan Finni"
    className="user-image"
  />
</div>
`;
